lib/kino/ets.ex

```diff
@@ -51,9 +49,9 @@ defmodule Kino.ETS do
   @impl true
   def get_data(rows_spec, state) do
     records = get_records(state.tid, rows_spec)
-    rows = Enum.map(records, &record_to_row/1)
+    rows = Enum.map(records, fn record -> %{fields: %{0 => inspect(record)}} end)
     total_rows = :ets.info(state.tid, :size)
-    columns = records |> Utils.Table.keys_for_records() |> Utils.Table.keys_to_columns()
+    columns = if records == [], do: [], else: [%{key: 0, label: "rows"}]
```

Comment:
Since we know the table structure, we can always show the expected columns :) I think the column should be singular, like item/value/entry/record, any preference?

```suggestion
    columns = [%{key: 0, label: "item"}]
```

Reply:
(we already use `:item` for Ecto results that are lists of values, we should make sure the wording is the same)

Reply:
I would prefer row in this case because it is the whole ETS row (not sure if it is the case for Ecto). :)

Reply:
I see, for Ecto it's only the case when doing like `select: user.name`, so it's different. Let's go with "row" then!

Reply:
I'm not a huge fan of showing the header without data, but no strong opinion. It's up to you!
`:item` sounds good and keeps consistency. Like it!

Reply:
My suggestion:

```suggestion
    columns = [%{key: 0, label: "row", type: "tuple"}]
```

Am I using type correctly @jonatanklosko ?

Reply:
@josevalim yup!

Reply:
@cristineguadelupe if we want to hide columns on no data, we should do it in the template directly, so this applies to all cases. @josevalim any preference?

Reply:
What about just show "No data" like we are doing for other tables?

Reply:
@cristineguadelupe exactly, my question is if we want to do that. If so, then we just change the condition on the front end, so we don't need to remember about it for every table.

Reply:
Gotcha! I think we should. Looks better, clearer, and consistent.

Reply:
Sorry for ruining the party but I don't think we should. If you make a SQL query, for example, even if it returns no results, I think knowing the column names and types is important information. Same for when showing data frames.

Reply:
Ok, I made the other changes and kept the header